src/map/entities/battleentity.cpp

```diff
@@ -803,35 +803,35 @@ void CBattleEntity::addEquipModifiers(std::vector<CModifier> *modList, uint8 ite
             int16 modAmount = GetMLevel() * modList->at(i).getModAmount();
             switch (modList->at(i).getModID())
             {
-                case Mod::DEF:
-                case Mod::MAIN_DMG_RATING:
-                case Mod::SUB_DMG_RATING:
-                case Mod::RANGED_DMG_RATING:
-                    modAmount *= 3;
-                    modAmount /= 4;
-                    break;
-                case Mod::HP:
-                case Mod::MP:
-                    modAmount /= 2;
-                    break;
-                case Mod::STR:
-                case Mod::DEX:
-                case Mod::VIT:
-                case Mod::AGI:
-                case Mod::INT:
-                case Mod::MND:
-                case Mod::CHR:
-                case Mod::ATT:
-                case Mod::RATT:
-                case Mod::ACC:
-                case Mod::RACC:
-                case Mod::MATT:
-                case Mod::MACC:
-                    modAmount /= 3;
-                    break;
-                default:
-                    modAmount = 0;
-                    break;
+            case Mod::DEF:
```

Comment:
Looks like visual studio changed whitespace. Let me know if you'd like me to change it back.

Reply:
¯\\\_\(._.)\_/¯

we have it both ways in diff places in core. we need to pick one then example it to the style guide.